### PR TITLE
Force-disable flag in experiments via cookies

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

File: src/experiments.js [hunk 1]
```diff
@@ -124,11 +124,21 @@ export function isExperimentOnAllowUrlOverride(win, experimentId) {
  * @return {boolean}
  */
 function calcExperimentOn(win, experimentId) {
-  const cookieFlag = getExperimentIds(win).indexOf(experimentId) != -1;
+  const experiments = getExperimentIds(win);
+
+  // Disabling cookie flag.
+  const disableFlag = experiments.indexOf('-' + experimentId) != -1;
```
Comment (lannka) [inline src/experiments.js]: so this is for browser to manually opt-out?

Comment (dvoytenko) [inline src/experiments.js]: This is for a developer to manually opt-out. Basically, "We just launched feature X. Our binary config as X:1". There's no way in this scenario to override X back to 0 manually w/o redeployment. This is useful when you want to check if indeed the launch caused the problem or not.

Comment (lannka) [inline src/experiments.js]: sounds useful

Comment (lannka) [inline src/experiments.js]: Using `AMP.toggleExperiment('xxx', false)` can manually opt-out for the current page view. I guess that's not enough for your use case? You want it to be persistent?

Comment (dvoytenko) [inline src/experiments.js]: Currently it doesn't. It doesn't disable, but removes from enabled :)

**Pull request merged**